django/contrib/gis/db/models/fields.py

```diff
@@ -37,7 +37,7 @@ def get_srid_info(srid, connection):
     """
     from django.contrib.gis.gdal import SpatialReference
 
-    global _srid_cache
+    global _srid_cache  # NOQA: F824
```

Comment:
Should we drop these instead given there is no reassignment?

```suggestion
```

Reply:
Right :facepalm: Updated.